ERR_STREAM_WRITE_AFTER_END on subscription creation (/listen)

Sorry - still hitting an issue, and the subscription does not actually get created. 

I see it posting back now (in ngrok)

POST /listen?validationToken=Validation%3a+Testing+client+application+reachability+for+subscription+Request-Id%3a+70d77e72-0725-bd6c-2ec5-e8a03f84257a HTTP/1.1
Host: localhost:3000
Content-Length: 0
Accept-Encoding: gzip
Content-Type: text/plain; charset=utf-8
X-Forwarded-For: 13.85.192.123
X-Forwarded-Proto: https
X-Original-Host: 6666d37d425c.ngrok.io


but then NPM dies

Express server listening on port 3000
GET / 304 3.022 ms - -
POST /listen?validationToken=Validation%3a+Testing+client+application+reachability+for+subscription+Request-Id%3a+38ef101b-fa0f-ca42-b0d8-498fdee9aa15 200 9.538 ms - 117
events.js:292
      throw er; // Unhandled 'error' event
      ^

Error [ERR_STREAM_WRITE_AFTER_END]: write after end
    at writeAfterEnd (_http_outgoing.js:668:15)
    at ServerResponse.end (_http_outgoing.js:788:7)
    at _callee$ (C:\temp\dotnet\webhook\routes\/listen.js:83:22)
    at tryCatch (C:\temp\dotnet\webhook\node_modules\@azure\msal-node\node_modules\regenerator-runtime\runtime.js:63:40)
    at Generator.invoke [as _invoke] (C:\temp\dotnet\webhook\node_modules\@azure\msal-node\node_modules\regenerator-runtime\runtime.js:293:22)
    at Generator.next (C:\temp\dotnet\webhook\node_modules\@azure\msal-node\node_modules\regenerator-runtime\runtime.js:118:21)
    at asyncGeneratorStep (C:\temp\dotnet\webhook\routes\listen.js:28:103)
    at _next (C:\temp\dotnet\webhook\routes\listen.js:30:194)
    at C:\temp\dotnet\webhook\routes\listen.js:30:364
    at new Promise (<anonymous>)
Emitted 'error' event on ServerResponse instance at:
    at writeAfterEndNT (_http_outgoing.js:727:7)
    at processTicksAndRejections (internal/process/task_queues.js:81:21) {
  code: 'ERR_STREAM_WRITE_AFTER_END'
}
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! microsoft-graph-webhook-nodejs@1.0.0 start: `mocha ./tests/confTest.js && babel-node --config-file ./babel.config.json ./index`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the microsoft-graph-webhook-nodejs@1.0.0 start script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\jrosen\AppData\Roaming\npm-cache\_logs\2020-12-09T16_37_32_060Z-debug.log

--- a/routes/listen.js
+++ b/routes/listen.js
@@ -80,7 +80,7 @@ listenRouter.post('/', async (req, res, next) => {
       status = 202;
     }
   }
-  res.status(status).end(http.STATUS_CODES[status]);
+  res.status(status).end();
 });
 
 function emitNotification(subscriptionId, data) {

--- a/routes/listen.js
+++ b/routes/listen.js
@@ -1,5 +1,4 @@
 import express from 'express';
-import http from 'http';
 import escape from 'escape-html';
 
 import { ioServer } from '../helpers/socketHelper';

